packages/marketplace/src/components/pages/developer-submit-app/developer-submit-app.tsx

```diff
@@ -280,11 +280,17 @@ export type HandleOpenAppPreview = {
   scopes: ScopeModel[]
   categories: CategoryModel[]
   values: FormikValues
-  appId?: string
+  appId: string
   appDetails?: AppDetailModel & { apiKey?: string }
 }
 
-export const handleOpenAppPreview = ({ appDetails, values, scopes, categories, appId }: HandleOpenAppPreview) => () => {
+export const handleOpenAppPreview = ({
+  appDetails,
+  values,
+  scopes,
+  categories,
+  appId = 'submit-app',
```

Comment:
why `appId` default is `submit-app`?

Reply:
@vuhuucuong because in case submit new app, there is no appId

Reply:
when creating a new app is haven't Id, So I set default ID is submit-app to detect that we are previewing a new app or existed app

Reply:
Got it, but I think better to add a comment to avoid confusing